spec/models/portfolio_item_spec.rb

```diff
@@ -49,4 +49,14 @@
       expect(PortfolioItem.policy_class).to eq(PortfolioItemPolicy)
     end
   end
+
+  context "callbacks" do
+    %i[create destroy discard undiscard].each do |kind|
+      it "calls update_portfolio_stats on #{kind}" do
+        expect(subject).to receive(:update_portfolio_stats)
```

Comment:
This one too, should make sure that the `portfolio` is receiving the correct message, not the object under test.